docs/deployment.md

```diff
@@ -0,0 +1,16 @@
+# Deployment From PR to Production
+
+1. Create a PR. Get it reviewed.
+2. **Make sure master is merged into the PR branch** and make sure all CI tests pass.
+3. Deploy to your QA slack and app environment first. (Something like `git push qa my_pr_branch:master`)
```

Comment:
I feel like something more is needed before 3 in order to make sure that QA is actually first in the same state as PROD master - especially as it involves migrations.

Reply:
Yeah I'd love to see more exact commands. For example step 1: `export MY_BRANCH=<your-branch-name>` Then your commands can be copy-pasted exactly, `git push qa $MY_BRANCH:master`

Reply:
added